.github/workflows/release_mac.yml

```diff
@@ -0,0 +1,70 @@
+name: MacRelease
+
+on:
+  push:
+    branches:
+      - rel-*
+  # test this PR for now, remove it before the merge
+  pull_request:
+    branches: [ master ]
+env:
+  MACOSX_DEPLOYMENT_TARGET: 10.12
+jobs:
+  build:
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+        architecture: ['x64']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+        architecture: ${{ matrix.architecture }}
+    - name: Install Python dependencies
+      run: |
+        python -m pip install -q --upgrade pip
+        python -m pip install -q numpy setuptools wheel
+    - name: Install protobuf dependencies
+      run: |
+        # Install protobuf 3.11.3
+        brew update
+        brew install autoconf && brew install automake
+        cd ..
+        wget https://github.com/protocolbuffers/protobuf/releases/download/v3.11.3/protobuf-cpp-3.11.3.tar.gz
+        tar -xvf protobuf-cpp-3.11.3.tar.gz
+        cd protobuf-3.11.3
+        ./autogen.sh
+        ./configure --disable-shared
+        make
+        make check
+        make install
+        cd ../onnx
+    - name: Build wheel and install
+      env:
+        ONNX_NAMESPACE: "ONNX_REL_1_8_1"
```

Comment:
What is the significance of setting the namespace to this? Isn't default namespace (onnx) ok? If we use release specific namespace then we will have to update this CI for every release. 1 more task for release manager

Reply:
We have a similar discussion before. IMO, not setting it should be fine. Just removed. Thanks for the reminder.